icu4c/source/configure

```diff
@@ -1109,6 +1114,15 @@ do
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
```

Comment:
^ This 'runstatedir' stuff probably comes because you are using an unreleased `autoconf`. (2.70 still is not released) Just ignore it, I guess.